test-lib.sh

```diff
@@ -518,6 +558,10 @@ EOF
     fi
     # Filter out env var definitions from $s2i_args and create Dockerfile ENV commands out of them
     echo "$s2i_args" | grep -o -e '\(-e\|--env\)[[:space:]=]\S*=\S*' | sed -e 's/-e /ENV /' -e 's/--env[ =]/ENV /' >>"$df_name"
+    mount_ca_option=$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]/etc/pki/\.*\S*')
```

Comment:
Again, please make it general. This still only looks for volume arguments that contain `/etc/pki/`